vue-app/src/views/FactoryRound.vue

```diff
@@ -0,0 +1,771 @@
+<template>
+  <div id="app" class="wrapper">
+    <nav-bar :inApp="isInApp" />
+    <div id="content-container">
+      <div
+        id="sidebar"
+        v-if="isSidebarShown"
+        :class="`${$store.state.showCartPanel ? 'desktop-l' : 'desktop'}`"
+      >
+        <round-information />
+      </div>
+      <div
+        id="content"
+        :class="{
+          padded: isSidebarShown && !isCartPadding,
+          'mr-cart-open': isCartToggledOpen && isSideCartShown,
+          'mr-cart-closed': !isCartToggledOpen && isSideCartShown,
+        }"
+      >
+        <back-link
+          v-if="showBackLink"
+          :to="backLinkRoute"
+          :text="backLinkText"
+        />
+        <router-view :key="$route.path" />
+      </div>
+      <div
+        id="cart"
+        v-if="isSideCartShown && !isHistoricRound"
+        :class="`desktop ${isCartToggledOpen ? 'open-cart' : 'closed-cart'}`"
+      >
+        <cart-widget />
+      </div>
+    </div>
+    <mobile-tabs v-if="isMobileTabsShown" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch } from 'vue-property-decorator'
+
+import { getCurrentRound } from '@/api/round'
+import { User } from '@/api/user'
+
+import RoundInformation from '@/views/RoundInformation.vue'
+import NavBar from '@/components/NavBar.vue'
+import CartWidget from '@/components/CartWidget.vue'
+import MobileTabs from '@/components/MobileTabs.vue'
+import BackLink from '@/components/BackLink.vue'
+
+import {
+  LOAD_USER_INFO,
+  LOAD_ROUND_INFO,
+  LOAD_RECIPIENT_REGISTRY_INFO,
+  SELECT_ROUND,
+  LOAD_CART,
+  LOAD_COMMITTED_CART,
+  LOAD_CONTRIBUTOR_DATA,
+  LOGIN_USER,
+} from '@/store/action-types'
+import {
+  SET_CURRENT_USER,
+  SET_ACTIVE_ROUND_ADDRESS,
+  TOGGLE_SHOW_CART_PANEL,
+} from '@/store/mutation-types'
+
+@Component({
+  name: 'clr.fund',
+  metaInfo: {
+    title: 'clr.fund',
+    titleTemplate: 'clr.fund - %s',
+    meta: [
+      {
+        name: 'git-commit',
+        content: process.env.VUE_APP_GIT_COMMIT || '',
+      },
+    ],
+  },
+  components: {
+    RoundInformation,
+    NavBar,
+    MobileTabs,
+    CartWidget,
+    BackLink,
+  },
+})
+export default class FactoryRound extends Vue {
```

Comment:
Small nitpick - I don't love the name `FactoryRound`... if anything I'd go with `RoundFactory` (which I don't love either) to keep more consistent with the `FundingRoundFactory` contract?